selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -140,6 +144,16 @@ public ProjectExplorer(
   @FindBy(xpath = Locators.PROJECT_EXPLORER_TAB_IN_THE_LEFT_PANEL)
   WebElement projectExplorerTabInTheLeftPanel;
 
+  /**
+   * @param path path to item in Project Explorer
+   * @return item by path
+   */
+  private WebElement getProjectExplorerItem(String path) {
+    String locator = "//div[@path='/%s']/div";
```

Comment:
This variable seems is redundant here. Let's get rid of it to simplify the code.

Reply:
We use this method 7 times, so on my opinion, this is better apply this method.